bin/strata-cli/src/net_type.rs

```diff
@@ -1,7 +1,7 @@
 use std::str::FromStr;
 
 /// Represents a type of network, either Alpen's signet or Strata
-#[derive(PartialEq, Eq)]
+#[derive(PartialEq, Eq, Debug, Clone)]
```

Comment:
This should be `#[non_exhaustive]` since we might add Testnet and Bitcoin in the future 🚀